src/cli.ts

```diff
@@ -50,6 +57,10 @@ async function main(): Promise<void> {
         ? cliConfig.backgroundPath
         : undefined,
       foregroundPath: cliConfig.foregroundPath
+    },
+    exclude: {
+      android: cliConfig.excludePlatforms.includes("android"),
```

Comment:
While the documentation mentions only lowercase variants as allowed values, it might be safer to do a case insensitive match.

Alternatively, if there is a good reason for the values to be case sensitive, it might be good to state it in the documentation.

Reply:
Sure, any ideas on the best way to do this? I don't think it works with `Array.includes()`.

Reply:
Only way I can think of is like this (`excludePlatforms` negation removed), but that doesn't seem very clean 🤔
```ts
platforms: {
  android: !!cliConfig.platforms.find((e) => e.toLowerCase() === "android"),
  ios: !!cliConfig.platforms.find((e) => e.toLowerCase() === "ios")
}
```

Reply:
I would maybe have a separate normalisation step for the `plaforms` value, which would produce an array of lowercase values. Then the comparisons would just be the same as before.